.github/scripts/get-excluded-crates.sh

```diff
@@ -0,0 +1,49 @@
+CRATES="protocol oauth core discovery audio metadata playback connect"
+
+CURRENT_TAG=$(git describe --abbrev=0)
+LAST_TAG=$(git describe --abbrev=0 $CURRENT_TAG^)
+
+BIN_VERSION=$(cat ./Cargo.toml | awk "/version/{print; exit}" | cut -d\" -f 2)
+
+SIMPLE_VERSION_REGEX="^v?([0-9]+)\.([0-9]+)\.([0-9]+)$"
```

Comment:
This one should match special versions (e.g., v1.0.0-beta.1, v1.0.0-rc.1) and those with build metadata(e.g., v1.0.0+20130313144700), and _not_ match invalid versions like v999999999999999999.0.0:
 
```shell
SEMVER_REGEX="^v?([0-9]+)\.([0-9]+)\.([0-9]+)(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$"
```

Please verify it though 😄

Reply:
Do we intend to use pre-release versions? I wanted to keep the regex as simple as possible as otherwise the whole logic around what version jump we made would explode.

Reply:
We haven't yet. So OK by not supporting it for now.